pkgs/by-name/qr/qrencode/package.nix

```diff
@@ -20,21 +22,32 @@ stdenv.mkDerivation (finalAttrs: rec {
   ];
 
   src = fetchurl {
```

Comment:
Use fetchFromGitHub instead.

Reply:
Thanks for the suggestion. I'll update the PR soon.